rfc/rfc005-distributed-network-using-grpc.md

```diff
@@ -115,7 +115,10 @@ If the response message exceeds the maximum Protobuf message size (as defined by
 
 ### 5.4. Resolving Transaction Content
 
-When the local node is missing a transaction's content, it SHOULD query the peer that provided the transaction for the content using the `TransactionPayloadQuery` message. The peer MUST respond with the `TransactionPayload` message, providing the actual content in the `data` field. If the peer doesn't have the content the `data` field MUST be left empty.
+When the local node is missing a transaction's content, it SHOULD query the peer that provided the transaction for the content using the `TransactionPayloadQuery` message.
+The peer MUST respond with the `TransactionPayload` message, providing the actual content in the `data` field. If the peer doesn't have the content the `data` field MUST be left empty.
+It MAY NOT query the peer for the content if the transaction contains the `to` field. Transactions with this field can only be queried with higher level protocol versions.
```

Comment:
What does "It" refer to here (I assume "the local node")? I'd name it explicitly.

> with higher level protocol versions
 "higher level" sounds a bit weird, "newer protocols", or just "higher protocol versions"

If the peer MUST respond with an empty `data` field for these TXs we can say the peer "SHOULD NOT" query it?

Reply:
adjusted the first 2. The last point is because evil nodes might still try it and an answer should be defined in that case.